slack-archive/src/test/java/juja/microservices/slack/utils/Util.java

```diff
@@ -0,0 +1,23 @@
+package juja.microservices.slack.utils;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+
+public class Util {
+
+    public static String readStringFromFile(String fileName){
+
+        String result = "";
+
+        Foo foo = new Foo();
```

Comment:
the strange logic for reading the file
For example, you can use in test
`IOUtils.toString(this.getClass().getClassLoader().getResourceAsStream(fileName));`

Reply:
the method is static, I can't use non static method this.getClass() here

Reply:
I want to return this code into test or another case classz should be ones from parameters this method
After that this method becomes simple